libraries/AP_BattMonitor/AP_BattMonitor_INA2xx.cpp

```diff
@@ -146,11 +145,33 @@ bool AP_BattMonitor_INA2XX::write_word(const uint8_t reg, const uint16_t data) c
 
 void AP_BattMonitor_INA2XX::timer(void)
 {
+    // allow for power-on after boot
+    if (!configured) {
+        uint32_t now = AP_HAL::millis();
```

Comment:
```suggestion
        const uint32_t now = AP_HAL::millis();
```
Nitpick, feel free to ignore.